swarm/api/http/handler.go

```diff
@@ -0,0 +1,120 @@
+package http
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/ethereum/go-ethereum/swarm/log"
+)
+
+func (s *Server) HandleBzzGet() http.HandlerFunc {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		log.Debug("handleGetBzz")
+		if r.Header.Get("Accept") == "application/x-tar" {
+			uri := GetURI(r.Context())
+			reader, err := s.api.GetDirectoryTar(r.Context(), uri)
+			if err != nil {
+				RespondError(w, r, fmt.Sprintf("Had an error building the tarball: %v", err), http.StatusInternalServerError)
+			}
+			defer reader.Close()
+
+			w.Header().Set("Content-Type", "application/x-tar")
+			w.WriteHeader(http.StatusOK)
+			io.Copy(w, reader)
+			return
+		}
+
+		s.HandleGetFile(w, r)
+	})
+}
+
+func (s *Server) HandleBzzPost() http.HandlerFunc {
```

Comment:
If this function just wraps one log line, why that log line is not in the HandlePostFiles, completely removing the need for HandleBzzPost?

The same goes to all functions that wrap one log line in this file.